src/features/game/expansion/lib/expansionNodes.ts

```diff
@@ -10,6 +10,7 @@ export interface Nodes {
   "Stone Rock": number;
   "Iron Rock": number;
   "Gold Rock": number;
+  "Ruby Rock": number;
```

Comment:
These data structures are used for lands that haven't prestiged yet so they shouldn't have Ruby Rock, right?

Reply:
Yep, these are for testing.. 
Draft PR.